service/logging_config.py

```diff
@@ -0,0 +1,20 @@
+import logging
+from logging import config
+import json
+from config import CONFIG_DICT
+
+done_setup = False
+
+
+def setup_logging():
```

Comment:
Moved this method into a separate module, because now there are two places that execute it. Alternatively, `gunicorn_settings` could just import `service` package, but the purpose of doing so would be very unclear and someone could just remove the import without being aware of breaking application event logging.